crates/rpc/rpc-types-compat/src/engine/payload.rs

```diff
@@ -360,6 +360,44 @@ pub fn convert_to_payload_body_v1(value: Block) -> ExecutionPayloadBodyV1 {
     ExecutionPayloadBodyV1 { transactions: transactions.collect(), withdrawals: withdraw }
 }
 
+/// Transforms a [reth_primitives::BlobTransactionSidecar] into a [reth_rpc_types::BlobTransactionSidecar]
+pub fn from_primitive_sidecar(sidecar: reth_primitives::BlobTransactionSidecar) -> reth_rpc_types::BlobTransactionSidecar {
+    reth_rpc_types::BlobTransactionSidecar {
+        blobs: sidecar.blobs,
+        commitments: sidecar.commitments, 
+        proofs: sidecar.proofs,
+    }
+}
```

Comment:
why do we have this twice?

Reply:
There was already an old one elsewhere—removing